packages/react-components/components/Button.v2.tsx

```diff
@@ -60,7 +62,7 @@ export default React.memo(function Button(props: ButtonProps) {
   return (
     <View style={getStyleForWrapper(size, style)}>
       {/* these Views cannot be combined as it will cause ripple to not respect the border radius */}
-      <View style={styles.containRipple}>
+      <View style={[styles.containRipple, rounded && { borderRadius: 100 }]}>
```

Comment:
Should we move this inline borderRadis style obj to `styles`?

Reply:
Yes thanks!